modules/pages/client/components/Faq.component.js

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Board from '@/modules/core/client/components/Board.js';
+import { Trans, useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+
+export default function Faq({ category, invitationsEnabled, children }) {
+  const { t } = useTranslation('pages');
+
+  // TODO connect to the API and remove mock data
+  const faq = {
+    allowStickySidebar: false,
+    waypoints: {
+      flags: {
+        on: true,
+      },
+    },
+  };
+
+  return (
+    <>
+      <Board className="faq-header" names={['happyhippies', 'guitarcamper']}>
+        <div className="container">
+          <div className="row">
+            <div className="col-xs-12 text-center">
+              <h2 className="visible-xs-block">{t('FAQ')}</h2>
+              <h2 className="hidden-xs">{t('Frequently Asked Questions')}</h2>
+              {category === 'general' && (
+                <h3>{t('about the site & community')}</h3>
+              )}
+              {category === 'tribes' && <h3>{t('about tribes')}</h3>}
+              {category === 'foundation' && (
+                <h3>{t('about the foundation')}</h3>
+              )}
+              {category === 'technology' && <h3>{t('about technology')}</h3>}
+            </div>
+          </div>
+        </div>
+      </Board>
+
+      <section className="container">
+        <div className="row">
+          <div className="col-xs-12 col-sm-4 col-md-4">
+            <div
+              id="faq-sidebar"
+              className={classnames('sidebar', {
+                'sidebar-sticky':
+                  faq.waypoints.flags.on && faq.allowStickySidebar,
+              })}
+              // zum-waypoint="faq.waypoints" // TODO
+              // down="flags.on" // TODO
+              // up="flags.off" // TODO
```

Comment:
How should I implement these?

Reply:
I think it's fine if we dont. It's not very important feature.

Reply:
Alright, I took it out

Reply:
Thanks for review!